packages/vscode-extension/src/officeDevHandlers.ts

```diff
@@ -0,0 +1,218 @@
+  );
+}
+
+export async function openDocumentHandler(args?: any[]): Promise<Result<boolean, FxError>> {
+  return VS_CODE_UI.openUrl("https://learn.microsoft.com/office/dev/add-ins/");
+}
+
+export async function openDevelopmentLinkHandler(args: any[]): Promise<Result<boolean, FxError>> {
+  return VS_CODE_UI.openUrl(
+    "https://learn.microsoft.com/office/dev/add-ins/develop/develop-overview"
+  );
+}
+
+export async function openLifecycleLinkHandler(args: any[]): Promise<Result<boolean, FxError>> {
+  return VS_CODE_UI.openUrl(
+    "https://learn.microsoft.com/office/dev/add-ins/overview/core-concepts-office-add-ins"
+  );
+}
+
+export async function openHelpFeedbackLinkHandler(args: any[]): Promise<Result<boolean, FxError>> {
+  return VS_CODE_UI.openUrl("https://learn.microsoft.com/answers/tags/9/m365");
+}
+
+export async function openReportIssues(args?: any[]): Promise<Result<boolean, FxError>> {
+  return VS_CODE_UI.openUrl("https://github.com/OfficeDev/office-js/issues");
+}
+
+export function validateOfficeAddInManifest(args?: any[]): Promise<Result<null, FxError>> {
+  const terminal = OfficeDevTerminal.getInstance();
+  terminal.show();
+  terminal.sendText(triggerValidate);
+  return Promise.resolve(ok(null));
+}
+
+export function installOfficeAddInDependencies(args?: any[]): Promise<Result<null, FxError>> {
+  const terminal = OfficeDevTerminal.getInstance();
+  terminal.show();
+  terminal.sendText(triggerInstall);
+  return Promise.resolve(ok(null));
+}
+
+export async function popupOfficeAddInDependenciesMessage() {
+  const buttonOptions = ["Yes", "No"];
+  const notificationMessage = localize("teamstoolkit.handlers.askInstallOfficeAddinDependency");
+
+  const result = await vscode.window.showInformationMessage(notificationMessage, ...buttonOptions);
+
+  if (result === "Yes") {
+    // Handle Yes button click
+    await autoInstallDependencyHandler();
+  } else if (result === "No") {
+    // Handle No button click
+    void vscode.window.showInformationMessage(
+      localize("teamstoolkit.handlers.installOfficeAddinDependencyCancelled")
+    );
+  } else {
+    // Handle case where pop-up was dismissed without clicking a button
+    // No action.
+  }
+}
+
+export function stopOfficeAddInDebug(args?: any[]): Promise<Result<null, FxError>> {
+  const terminal = OfficeDevTerminal.getInstance();
+  terminal.show();
+  terminal.sendText(triggerStopDebug);
+  return Promise.resolve(ok(null));
+}
+
+export function generateManifestGUID(args?: any[]): Promise<Result<null, FxError>> {
+  const terminal = OfficeDevTerminal.getInstance();
+  terminal.show();
+  terminal.sendText(triggerGenerateGUID);
+  return Promise.resolve(ok(null));
+}
+
+export function editOfficeAddInManifest(args?: any[]): Promise<Result<null, FxError>> {
+  const workspacePath = globalVariables.workspaceUri?.fsPath;
+  if (!workspacePath) {
+    void VS_CODE_UI.showMessage("error", `Not valid workspace path`, false);
```

Comment:
Please also use localized string in popup messages.

Reply:
@tecton Fixed.